qubes/tests/integ/dispvm_perf.py

```diff
@@ -92,141 +107,169 @@ def run_test(self, name):
         if p.returncode:
             self.fail(f"'{' '.join(cmd)}' failed: {p.returncode}")
 
-    def test_000_dispvm(self):
+    def run_reader(self, args: list):
+        results = os.getenv("QUBES_TEST_PERF_FILE")
+        if not results:
+            self.fail("Did not set QUBES_TEST_PERF_FILE")
+        cmd = [
+            "/usr/lib/qubes/tests/dispvm_perf_reader.py",
+            f"--output-dir={self.test_dir}",
```

Comment:
Teardown will remove this dir. Can you make it at least configurable to leave it in place in some dir? Or maybe even always output to some specific dir (name based on `QUBES_TEST_PERF_FILE` or something)? Then I can make openQA to actually collect rendered graphs